.github/workflows/wheels.yml

```diff
@@ -6,15 +6,15 @@ on:
 
 jobs:
   sdist:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     - name: Set up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v4
       with:
-        python-version: 3.9
+        python-version: "3.10"
```

Comment:
To avoid updating this line every year, this could be `"3.x"`, which will be a recent, stable version.

It's currently 3.10, and won't be switched to 3.11 for a little while.